tests/QueuedExportTest.php

```diff
@@ -48,30 +43,6 @@ public function can_queue_export_with_remote_temp_disk()
     {
         config()->set('excel.temporary_files.remote_disk', 'test');
 
-        // Delete the local temp file before each append job
```

Comment:
Removed this test sequence since it's obsole. The payload doesn't hold the local temporary file.